test/txsim/run_test.go

```diff
@@ -170,6 +170,8 @@ func TestTxSimUpgrade(t *testing.T) {
 		WithFundedAccounts("txsim-master")
 	cctx, _, grpcAddr := testnode.NewNetwork(t, cfg)
 
+	time.Sleep(time.Second * 2)
```

Comment:
Why was this sleep added. Could we instead just use the cctx to wait for the first block to be produced?

Reply:
forget which commit this was fixed, but it was fixed. sometimes it still flakes